programs/ssl/ssl_base64_dump.c

```diff
@@ -248,14 +254,55 @@ size_t read_next_b64_code( uint8_t *b64, size_t max_len )
     return 0;
 }
 
+void print_deserialized_ssl_cert( const uint8_t *ssl, size_t len )
+{
+    enum { STRLEN = 4096 };
```

Comment:
Is this going to be long enough? Could we allocate the exact right amount with malloc() instead?

Reply:
Unfortunately, I don't see a function that would check the required buffer size for a given certificate, but during my tests the output strings had around 300 characters, so I think 4096 is much more than required.

Reply:
the second thought is that `mbedtls_x509_crt_info` knows the size of the input buffer and it should be safe for memory